lib/colors.js

```diff
@@ -30,28 +31,57 @@ const colored = {
   greyMessage: gray
 }
 
-function colorizeLevel (level, colorizer) {
+function resolveCustomColored (customColors) {
```

Comment:
Is this supposed to be?:

```suggestion
function resolveCustomColors (customColors) {
```